packages/ckeditor5-core/src/editor/editor.js

```diff
@@ -433,6 +458,21 @@ export default class Editor {
 		this.editing.view.focus();
 	}
 
+	/**
+	 * Generates a label for the editor.
+	 *
+	 * It will be based on the editor.config.get( 'label' ), otherwise a name
+	 * pattern of `'editor{n}'` will be used.
+	 *
+	 * @readonly
+	 * @returns {String} label
+	 */
+	generateEditorLabel() {
+		const label = this.config.get( 'label' ) ? this.config.get( 'label' ) : 'editor' + Editor.editorCount;
```

Comment:
There's no documentation for `label` in `EditorConfig`. This a critical API for integrators and must be documented.